kjdoc/tour.md

```diff
@@ -538,13 +538,440 @@ This section describes KJ APIs that control process execution and low-level inte
 
 KJ makes asynchronous programming manageable using an API modeled on E-style Promises. E-style Promises were also the inspiration for JavaScript Promises, so modern JavaScript programmers should find KJ Promises familiar, although there are some important differences.
 
-A `kj::Promise<T>` represents an asynchronous background task that, upon completion, will either "resolve" to a value of type `T`, or "reject" with an exception.
+A `kj::Promise<T>` represents an asynchronous background task that, upon completion, either "resolves" to a value of type `T`, or "rejects" with an exception.
```

Comment:
I don't know if it might be worth calling out the third possibility of promise cancellation here?

Reply:
Eh, I think that's better introduced separately.